cartographer/sensor/proto/sensor.proto

```diff
@@ -20,6 +20,15 @@ option java_outer_classname = "Sensor";
 
 import "cartographer/transform/proto/transform.proto";
 
+message RangefinderPoint {
+  transform.proto.Vector3f position = 1;
+}
+
+message TimedRangefinderPoint {
+  RangefinderPoint rangefinder_point = 1;
```

Comment:
inline the type: `transform.proto.Vector3f point = 1;` . It can happen that `RangefinderPoint` has some metadata that we do not really want in `TimedRangefinderPoint`.

Reply:
Okay, should the C++ implementation follow this as well? I.e. `TimedRangefinderPoint` contains a `Vector3f`?

Reply:
Yep